woocommerce/woocommerce-admin
# Remove OnboardingEmailMarketing note class

Fixes #7514 

This PR removes `OnboardingEmailMarketing` note class.

### Detailed test instructions:

1. Checkout this branch.
2. Install and activate [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/) plugin.
3. Delete `wc-admin-onboarding-email-marketing` note from `wp_wc_admin_notes`. You can also use `delete from wp_wc_admin_notes where name='wc-admin-onboarding-email-marketing'` query.
4. Navigate to Tools -> Cron Events and run `wc_admin_daily`
5. Check `wp_wc_admin_notes` and make sure `wc-admin-onboarding-email-marketing` has not been re-created.
